staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -125,8 +125,7 @@ type configController struct {
 	requestWaitLimit time.Duration
 
 	// This must be locked while accessing flowSchemas or
-	// priorityLevelStates.  It is the lock involved in
-	// LockingWriteMultiple.
```

Comment:
The remark
> It is the lock involved in LockingWriteMultiple
got misplaced in past refactoring.  It belongs on the lock in `queueSet` and refers to the promise in every `request` in the queueSet.  It wouldn't hurt to reiterate the point in the comment on the `decision` field of `request`.

Reply:
I went step further and simplified it further (we don't really take advantage of "locking" part of the interface anywhere).

So to make it clear for everyone - I updated the interface itself to not be thread-safe [which is how it is used - we care about protecting it ourselves in queueset]